build.bat

```diff
@@ -1,25 +1,48 @@
 @echo off
 setlocal EnableDelayedExpansion
 pushd %~dp0
-IF "%1"=="" (set COVERAGE="--no-coverage") ELSE (set COVERAGE=%1)
-set HAZELCAST_VERSION=3.7-SNAPSHOT
+
+IF -%1-==-- (set COVERAGE="--no-coverage") ELSE (set COVERAGE=%1)
```

Comment:
I think this style of parameter passing will be problematic, if you specify for example build.bat --enterprise